person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, name, age, parents_permission = 'Unknown')
+    @id = id,
+          @name = name,
+          @age = age,
+          @parents_permission = parents_permission
+  end
+
+  private
+
+  def of_age?
+    if @age >= 18
+      puts true
+    else
+      puts false
+    end
+  end
+
+  def can_use_services?
+    return unless @age >= 18 || @parents_permission == true
+
+    puts true
```

Comment:
- After making the suggested changes in the `of_age` method, I suggest calling the method in the `can_use_services` method i.e `of_age?() || @parent_permission == true`